### DISCUSSION TITLE
Does the RHF supports multiple validation mode for any field?

### DISCUSSION THREAD

**Discussion by kmfb** [react-hook-form/react-hook-form#9061] [Q&A] [answered]
Since the react hook form set the Form Validation mode in the useForm's config, it can validate the whole form by the mode. But in some scenes, one of the fields need to be validated by multiple mode or other mode (not like the whole form config).

For example, the phone number can be validated by RE pattern in onChange mode, and it can be validated by calling back end api, which is should be onBlur mode. So I can't find any way to support this scene.

Dose anyone knows how to handle this problem? Thanks.

**Comment by bluebill1049** [comment, marked as answer]
https://github.com/react-hook-form/react-hook-form/issues/9064#issuecomment-1250853355
